src/components/KSelect/KSelect.vue

```diff
@@ -255,6 +257,13 @@ export default defineComponent({
       type: Function,
       default: (params: SelectFilterFnParams) => params.items.filter((item: SelectItem) => item.label.toLowerCase().includes(params.query.toLowerCase())),
     },
+    /**
+     * Control whether the input for `select` and `dropdown` appearances supports filtering.
+     */
+    enableFiltering: {
+      type: Boolean,
+      default: null,
```

Comment:
issue: the `default` would need to be `true` or `false` since `null` is not a valid Boolean value.

Reply:
TS actually accepts this value and I need it to be `null` because I need to know if they explicitly set it to either `true` or `false` as these should override the default values (which differ based on the `appearance`).

![image](https://user-images.githubusercontent.com/67973710/179234764-3f92c45f-2151-4808-ad15-f40afc7b182a.png)

https://fettblog.eu/boolean-in-javascript-and-typescript/